ws-nextjs-app/pages/[service]/new_live/[id]/KeyPoints/types.ts

```diff
@@ -0,0 +1,29 @@
+import { OptimoBlock } from '#models/types/optimo';
+
+interface ListItems {
+  type: string;
+  model: { blocks: OptimoBlock[] };
+}
+
+interface SummaryUnorderedList {
+  type: string;
+  model: { blocks: ListItems[] | [] };
+}
+
+export interface SummaryListWrapper {
+  type: string;
+  model: { blocks: SummaryUnorderedList[] };
+}
```

Comment:
Is this SummaryListWrapper just a weird Optimo block model nested thing?

Reply:
I think I have improved this by updating the naming conventions.